src/lib/rc/dsm.cpp

```diff
@@ -56,12 +56,7 @@
 
 #include <include/containers/Bitset.hpp>
 
-#if defined(__PX4_NUTTX)
-#include <nuttx/arch.h>
-#define dsm_udelay(arg)    up_udelay(arg)
```

Comment:
Did you verify this isn't ever called form an interrupt context?

Reply:
I need to investigate this more, turns out I missed one use case when grepping the first time. @dagar I'm going to remove this for now, if CI passes can you please take another look at just the ADC work?

Reply:
Postponing this to a later PR.